google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/CompositeTracer.java

```diff
@@ -178,4 +178,11 @@ public void recordGfeMetadata(@Nullable Long latency, @Nullable Throwable throwa
       tracer.recordGfeMetadata(latency, throwable);
     }
   }
+
+  @Override
+  public void batchRequestThrottled(long throttledTimeMs) {
+    for (BigtableTracer tracer : bigtableTracers) {
+      tracer.batchRequestThrottled(throttledTimeMs);
+    }
+  }
```

Comment:
Now that BigtableTracer is no longer abstract, we dont have a safeguard that ensures the CompositeTracer correctly proxies newly added methods. I think we need a CompositeTracerTest that reflectively ensures this